### PR TITLE
Fix installer build

### PR DESCRIPTION
**Pull request by FraserGreenroyd**
Links with https://github.com/BHoM/BHoM_Installer/pull/125

### CODE REVIEW

File: IES_Engine/Convert/Environment/Space.cs [hunk 1]
```diff
@@ -86,7 +86,7 @@ public static List<string> ToIES(this List<Panel> panelsAsSpace, SettingsIES set
                 List<Point> v = p.Vertices();
                 v.RemoveAt(v.Count - 1); //Remove the last point because we don't need duplicated points
 
-                if (!p.NormalAwayFromSpace(panels, settingsIES.PlanarTolerance) && p.ConnectedSpaces[0] == panels.ConnectedSpaceName())
+                if (!p.NormalAwayFromSpace(panels, settingsIES.PlanarTolerance)/* && p.ConnectedSpaces[0] == panels.ConnectedSpaceName()*/)
```
Comment (FraserGreenroyd) [inline IES_Engine/Convert/Environment/Space.cs:89]:
```suggestion
                if (!p.NormalAwayFromSpace(panels, settingsIES.PlanarTolerance))
```

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required

**Comment by FraserGreenroyd** [comment]
@BHoMBot check copyright-compliance

**Review by jamesramsden-bh — APPROVED** [review]
Tested against testing procedure, no problems encountered.

**Pull request merged**